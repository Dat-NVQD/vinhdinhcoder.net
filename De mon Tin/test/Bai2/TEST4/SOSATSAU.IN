100
57880086831606418866
78796334904195342418
28624965447014867946
08799592863373666254
12682294610713049538
