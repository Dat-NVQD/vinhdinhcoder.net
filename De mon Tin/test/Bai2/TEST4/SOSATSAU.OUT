5788008683160641886678796334904195342418286249654470148679460879959286337366625412682294610713049583
